.maintain/playbooks/templates/nginx/dali-stage.conf.j2

```diff
@@ -30,4 +30,88 @@ server {
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
     }
+
+    location /composable {
```

Comment:
logically sounds right, so I would prefer it would be nix derivation https://nixos.wiki/wiki/Nginx